packages/common-utils/src/queryParser.ts

```diff
@@ -531,7 +531,14 @@ export class CustomSchemaSQLSerializerV2 extends SQLSerializer {
       throw new Error('Unsupported column type for prefix match');
     }
 
-    throw new Error(`Column not found: ${field}`);
+    // It might be an alias, let's just try the column
+    // TODO: Verify aliases
```

Comment:
I'm not sure if this would cause weird behaviors (things like sql injection). Also, the error message is different if users search with an unexisting field. I think we can ticket it and move on. I just want to keep it on the radar.

Reply:
I don't think the issue is any worse than the existing SQL query where conditions? but agreed we should improve this later, but it doesn't feel material now

Reply:
nothing material atm. comments are helpful already